### ISSUE TITLE
No module named testsrunner

### ISSUE THREAD

**Issue by durack1** [PCMDI/cmor#671]
This issue appears with the CMOR 3.7.0 conda build against py3.10

`cmor               conda-forge/linux-64::cmor-3.7.0-py310h4b0e41b_0`

Email trail below
```
From: Gary Strand
Date: Thursday, August 25, 2022 at 9:42 AM
To: List for CMOR users
Subject: Failed to build CMOR3.7

I'm attempting to build the latest release of CMOR and am getting a failure:

[...]
Cloning into 'cmor'...
Submodule 'cmip6-cmor-tables' ([https://github.com/PCMDI/cmip6-cmor-tables.git](https://urldefense.us/v3/__https:/github.com/PCMDI/cmip6-cmor-tables.git__;!!G2kpM7uM-TzIFchu!i5gyuWLuW0OGfzDatLq3Ueh5_Fd5jPZ4O_C1IVIDRMC8aMeuD2g7v4lD7w84Rw$)) registered for path 'cmip6-cmor-tables'
Cloning into 'cmip6-cmor-tables'...
Submodule path 'cmip6-cmor-tables': checked out '4c999fab7e0a2aeb3c6597cf85501242c52fb573'
Traceback (most recent call last):
  File "/glade/u/home/strandwg/proj/software/cmor/CMIP6_work/cmor/run_tests.py", line 3, in <module>
    from testsrunner import TestRunnerBase
ModuleNotFoundError: No module named 'testsrunner'

Full output of build at

[https://drive.google.com/file/d/12612CbMVQPCNw5unwS_HPYH_ainfYPx4/view?usp=sharing](https://urldefense.us/v3/__https:/drive.google.com/file/d/12612CbMVQPCNw5unwS_HPYH_ainfYPx4/view?usp=sharing__;!!G2kpM7uM-TzIFchu!i5gyuWLuW0OGfzDatLq3Ueh5_Fd5jPZ4O_C1IVIDRMC8aMeuD2g7v4lsIwquAQ$)

Thanks for any guidance!

-- 
Gary Strand, CESM Data Manager
ORCID 0000-0001-9740-0104
strandwg@ucar
```

@strandwg ping

**Comment by durack1** [comment]
I'm wondering if https://github.com/CDAT/TestsRunner / https://anaconda.org/CDAT/testsrunner was a dependency which required the CDAT conda channel which was changed in https://github.com/PCMDI/cmor/pull/665

**Comment by mauzey1** [comment]
@durack1 Yes, this is similar to the issue encountered in #665, where conda was getting the outdated nightly version of CDMS2 rather than the one from conda-forge.

I have made a change in our source build instructions to address this problem.  I changed the order of conda channels from `-c cdat/label/nightly -c conda-forge -c cdat` to `-c conda-forge -c cdat/label/nightly -c cdat`.  It should now get the conda-forge build of CDMS2.
